css/style.css

```diff
@@ -72,7 +72,9 @@ header h2 {
 .content p {
     margin-bottom: 10px;
 }
-
```

Comment:
Should leave this blank line in, but below the new styles.

Reply:
Done. I also squashed because, come one, it's just 3 lines to review. :grin: